### PR TITLE
[screensaver.google.earth] 1.0.0

### PR DESCRIPTION
**Pull request by Lunatixz**
minor update

### Checklist:

- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-scripts/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [script.foo.bar] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practise but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-scripts/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.

### CODE REVIEW

File: screensaver.google.earth/gui.py [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+    def loadJson(self):
+        FILE = JSON_FILE_EXT if xbmcvfs.exists(JSON_FILE_EXT) else JSON_FILE
+        with open(FILE) as data: 
+            return json.load(data)
+     
+     
+    def prepareImages(self, results):
+        for img in results:
+            label = '%s, %s'%(img['country'],img['region']) if len(img['region']) > 0 else '%s'%(img['country'])
+            yield xbmcgui.ListItem(label,thumbnailImage=(img['image']))
```
Comment (learningit) [inline screensaver.google.earth/gui.py]: The thumbnailImage init is deprecated. Please use setArt instead.

Comment (Lunatixz) [reply screensaver.google.earth/gui.py]: I think that only applies to setThumbnailimage?

https://codedocs.xyz/xbmc/xbmc/group__python__xbmcgui__listitem.html#ga262604f2a38a80ef518a9662c45beb34

https://codedocs.xyz/xbmc/xbmc/group__python__xbmcgui__listitem.html#gac31a08def90f50295146753353cb9541

Comment (learningit) [reply screensaver.google.earth/gui.py]: I don't know. When I read this it looks pretty clear to me that it's deprecated:

http://mirrors.kodi.tv/docs/python-docs/16.x-jarvis/xbmcgui.html#ListItem

Comment (Lunatixz) [reply screensaver.google.earth/gui.py]: Yep those documents are clearer... I'll move to setart 👍

**Pull request merged**